frontend/frontendAngular/src/app/components/login/login.component.ts

```diff
@@ -0,0 +1,49 @@
+import { AuthenticationService } from './../../services/authentication.service';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { LoginRequest } from '../../api/authentication';
+import { LocalStorageLoginService } from '../../services/local-storage-login.service';
+
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+
+  form: FormGroup;
+  isLoading: boolean = false;
+  
+  constructor(public fb: FormBuilder, public loginService: AuthenticationService, private localStorage: LocalStorageLoginService) {
+    this.form = this.fb.group({
+      email: ['', [Validators.required, Validators.email]],
+      password: ['', [Validators.required]],
+    });
+  }
+
+  ngOnInit(): void {
+  }
+
+  submit(){
+    if (this.form.invalid){
+      this.form.markAllAsTouched();
+      return;
+    }
+
+    let request: LoginRequest = {
+      email: this.form.controls['email'].value,
+      password: this.form.controls['password'].value,
+    };
+    
+    this.isLoading = true;
+
+    this.loginService.login(request).subscribe((response) => {
+      console.log("Logged", response);
+      var obj = JSON.stringify(response, ['access']);
```

Comment:
creio q nao seja necessario usar o JSON.stringify... se vc for no loginService e definir o retorno do post `...post<LoginResponse>(...` criando a interface LoginResponse seja melhor